### DISCUSSION TITLE
Reopen fan control pull request

### DISCUSSION THREAD

**Discussion by rcdailey** [home-assistant/frontend#12071] [Dashboard]
A PR was automatically declined by a bot end of 2020 here:
https://github.com/home-assistant/frontend/pull/7803

Please reopen this PR for consideration. Had any decision been made regarding the slider control @bramkragten ?

**Comment by spacegaier** [comment]
As far as I know, the direction is still to get rid of the round slider, but no concrete timeline as of yet. I think I even saw a draft / PR here that contained a proposal for a replacement, with up/down buttons.

**Reply by rcdailey** [reply]
Shouldn't we just merge this and deal with the removal of the slider when that time comes? At this rate, years will go by before it gets addressed and all the while I won't have an aesthetically pleasing solution for variable-speed fan switches in lovelace. The ones I've seen in HACS are not what I would call "aesthetically pleasing". They're basically just a thin bar with a few buttons, but wouldn't look well next to my existing light buttons.
